src/eva/diagnostics/density.py

```diff
@@ -1,7 +1,7 @@
 from eva.eva_path import return_eva_path
 from eva.utilities.config import get
 from eva.utilities.utils import get_schema, update_object, slice_var_from_str
-import eva.plot_tools.plots
+from emcpy.plots.plots import Density as DensityPlot
```

Comment:
Is there a reason why not just keeping it as `emcpy.plots.plots.Density`? Or is that too verbose?

Reply:
I tried that, and it fails. I get the following error:
`ModuleNotFoundError: No module named 'emcpy.plots.plots.Scatter'; 'emcpy.plots.plots' is not a package`

Reply:
I'll try that.

Reply:
Thanks, I think that's clearer for the developers where the method comes from